code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -651,7 +651,7 @@
 	icon_state = "sleeper_s"
 	short_desc = "You are a syndicate operative on old ship, stuck in hostile space."
 	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Fix the ship, find a way to power it and follow your captain's orders."
-	important_info = "Obey orders given by your captain. DO NOT let ship fall into enemy's hands."
+	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy's hands."
```

Comment:
```suggestion
	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy hands."
```
Or `the enemy's`
Same for the other one.

Reply:
Already did this, isn't it?